### PR TITLE
Update aws-sam-translator Dependency

### PR DESCRIPTION
**Pull request by awood45**
The `make pr` command set is passing, so pending integration tests.

*Issue #, if available:* Numerous.

*Description of changes:* Bump aws-sam-translator dependency.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by awood45** [comment]
It does look like we need to add some new tests before merging this.

**Comment by awood45** [comment]
This now also addresses #803

File: samcli/commands/validate/lib/sam_template_validator.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def is_valid(self):
             template = sam_translator.translate(sam_template=self.sam_template,
                                                 parameter_values={})
             LOG.debug("Translated template is:\n%s", yaml_dump(template))
+            print("Translated template is:\n%s", yaml_dump(template))
```
Comment (jfuss) [inline samcli/commands/validate/lib/sam_template_validator.py]: remove print statement

**Comment by awood45** [comment]
This PR will resolve #1328 #1198

**Comment by beck3905** [comment]
@awood45 @jfuss Any idea when this will be released?

**Pull request merged**